packages/now-cli/package.json

```
@@ -41,7 +41,6 @@
     "all": true
   },
   "bin": {
```

Actually, we can specify the `"bin"` as a string and then it uses the field from `"name"` as the bin name. That way, the `utils/update-legacy-name.js` file doesn't need to be changed:

```suggestion
  "bin": "./dist/index.js"
```

---

Great idea! Fixed it 👍